cedar-policy-cli/src/lib.rs

```diff
@@ -118,17 +118,14 @@ pub enum Commands {
     New(NewArgs),
     /// Partially evaluate an authorization request
     PartiallyAuthorize(PartiallyAuthorizeArgs),
-    /// Ouput a JSON file for consumption by Lean
-    #[command(subcommand)]
-    WriteDRTJson(serialization::AnalysisCommands),
     /// Output a protobuf binary file for consumption by Lean
     #[cfg(feature = "protobufs")]
     #[command(subcommand)]
-    WriteDRTProto(serialization::AnalysisCommands),
+    WriteDRTProto(protobufs::AnalysisCommands),
```

Comment:
I guess I don't really know what protofbuf functionality was added to the CLI in the frist place, but how does the variant name (`WriteDRTProto`) relate to inner struct (`AnalysisCommands`)? Also, is this protobuf format still DRT specific, or should we rename this to something more general before release?

Reply:
The naming was done by #1326, I think the relationship is that `AnalysisCommands` is a list of the protobuf formats supported by `WriteDRTProto` (today, just `Equivalence`), all of which relate to analysis (at least today).  I agree this should be changed prior to release.  Let's track that in #1348; I see you've already linked this comment there